hacks.js

```diff
@@ -0,0 +1,17 @@
+// Addresses an issue where RTC peers are not properly garbage collected in Chromium:
+// https://bugs.chromium.org/p/chromium/issues/detail?id=825576
+// This error leads to WebTorrent Desktop and Chromium-based browsers being completely
+// unable to seed until the process is restarted:
+// https://github.com/webtorrent/webtorrent/issues/1981
+function forcePeerGC () {
+  queueMicrotask(() => {
+    let img = document.createElement('img')
+    img.src = window.URL.createObjectURL(new Blob([new ArrayBuffer(5e+7)]))
+    img.onerror = function () {
+      window.URL.revokeObjectURL(this.src)
+      img = null
+    }
```

Comment:
I still think this should only be ran if there have been peer connections closed since the last time this was ran, ideally this should be done in the simple-peer library, rather than webtorrent